core/Cargo.toml

```diff
@@ -21,6 +21,6 @@ std = ["math/std", "winter-utils/std", "winter-crypto/std"]
 
 [dependencies]
 math = { package = "winter-math", version = "0.4.2", default-features = false }
-crypto = { package = "miden-crypto", version = "0.1.0" }
+crypto = { package = "miden-crypto", git = "https://github.com/0xPolygonMiden/crypto.git", branch = "next"}
```

Comment:
I don't think this change should stay in the PR?

Reply:
This PR uses the hashing method `merge_in_domain` only available in the `next` branch of `miden-crypto`.   This is an interim solution to unblock development for this draft until a new release of `miden-crypto` is cut.  I will update to the appropriate version once the release has been made.